readme.md

```diff
@@ -75,6 +75,9 @@ This is a collection of **awesome resources** about [Prisma](https://www.prisma.
 ### Tutorials with [NestJS](https://nestjs.com/)
 - [How to query your database using Prisma with NestJS](https://notiz.dev/blog/how-to-connect-nestjs-with-prisma)
 
+## :building_construction: Community Examples
```

Comment:
We already have prisma-admin here: https://github.com/catalinmiron/awesome-prisma/pull/20/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R47

Reply:
@2color This is an example built with prisma-admin, not the [actual package](https://github.com/AhmedElywa/prisma-tools/tree/master/admin). :) However, it's probably a bit redundant so I'm closing the PR.